utils/_context/core.py

```diff
@@ -63,8 +63,10 @@ def __init__(self):
             self.waf_rule_set = Version("1.0.0")
         elif self.library == "java":
             self.waf_rule_set = Version("0.0.1")
+        elif self.library == "php":
+            self.waf_rule_set = Version("1.0.0")
         else:
-            self.waf_rule_set = Version("0.0.1")
+            self.waf_rule_set = Version("1.0.0")
```

Comment:
`else` old version :)

```suggestion
            self.waf_rule_set = Version("0.0.1")
```